pubsub/pulsar/metadata.yaml

```diff
@@ -0,0 +1,147 @@
+# yaml-language-server: $schema=../../../../component-metadata-schema.json
+schemaVersion: v1
+type: pubsub
+name: pulsar
+version: v1
+status: stable
+title: "Apache Pulsar"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-pubsub/setup-pulsar/
+authenticationProfiles:
+  - title: "Connection string"
+    description: "Authenticate using a Connection String"
+    metadata:
+      - name: host
+        type: string
+        required: true
+        description: "Address of the Pulsar broker."
+        example: |
+          "localhost:6650" OR "http://pulsar-pj54qwwdpz4b-pulsar.ap-sg.public.pulsar.com:8080"
+metadata:
+  - name: consumerID
+    type: string
+    description: "Used to set the subscription name or consumer ID."
+    example: "channel1"
+  - name: namespace
+    description: |
+      The administrative unit of the topic, which acts as a grouping mechanism for related topics.
+    type: string
+    default: '"default"'
+    example: '"default"'
+  - name: enableTLS
+    type: bool
+    description: |
+      Enable TLS.
+    default: 'false'
+    example: '"true", "false"'
+  - name: disableBatching
+    type: bool
+    description: |
+      When batching is enabled default batch delay is set to 10 ms and default
+      batch size is 1000 messages,Setting disableBatching: true will make the producer to send
+      messages individually.
+    default: 'false'
+    example: '"true", "false"'
+  - name: batchingMaxPublishDelay
+    type: duration
+    description: |
+      Sets the time period within which the messages sent will be batched,
+      if batch messages are enabled. If set to a non zero value, messages will be queued until this
+      time interval or batchingMaxMessages (see below) or batchingMaxSize (see below). There are two
+      valid formats, one is the fraction with a unit suffix format, and the other is the pure digital
+      format that is processed as milliseconds. Valid time units are “ns”, “us” (or “µs”), “ms”, “s”,
+      “m”, “h”.
+    default: '"10ms"'
```

Comment:
This double quotation is bad. If all of them need it then there is a problem in how these are consumed. I will not block this PR based on that since this seems to be a problem upstream.